modules/lang-painless/src/main/java/org/elasticsearch/painless/phase/PainlessUserTreeToIRTreePhase.java

```diff
@@ -425,30 +412,20 @@ protected void injectSandboxExceptions(FunctionNode irFunctionNode) {
 
             irInvokeCallNode.addArgumentNode(irLoadFieldMemberNode);
 
-            for (Class<?> throwable : new Class<?>[] {
+            irThrowNode = createCatchAndThrow(SecurityException.class, internalLocation, irTryNode);
```

Comment:
Can you update the comment in `injectSandboxExceptions` to document that `SecurityException` is caught and rethrown?